README.rst

```diff
@@ -148,21 +148,31 @@ the administration dashboard.
 Connection App
 ~~~~~~~~~~~~~~
 
-This app enables the controller to instantiate connections to the devices
+This app allows OpenWISP Controller to use different protocols to reach network devices.
+Currently, the default connnection protocols are SSH and SNMP, but the protocol
+mechanism is extensible and more protocols can be implemented if needed.
+
+SSH
+^^^
+
+The SSH connector allows the controller to initialize connections to the devices
 in order perform `push operations <#how-to-configure-push-updates>`__:
 
 - Sending configuration updates.
 - `Executing shell commands <#sending-commands-to-devices>`_.
 - Perform `firmware upgrades via the additional firmware upgrade module <https://github.com/openwisp/openwisp-firmware-upgrader>`_.
 
-The default connection protocol implemented is SSH, but other protocol
-mechanism is extensible and custom protocols can be implemented as well.
-
 Access via SSH key is recommended, the SSH key algorithms supported are:
 
 - RSA
 - Ed25519
 
+SNMP
+^^^^
+
+The SNMP connector is useful to collect monitoring information and it's used in
+`openwisp-monitoring`_.
```

Comment:
I improved this part and the one above a bit.

I suggest linking the SNMP section of the README in OpenWISP Monitoring so that readers are taken directly to the [instructions on how to use this](https://github.com/openwisp/openwisp-monitoring/pull/309#discussion_r692566202) (if they're interested  in it).